openshift/templates/kubernetes-network-policy/allow-5432.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Template
```

Comment:
maybe you should have called it something like allow-database-port-5432.yaml, at first i thought it was a new hip restaurant name.

Reply:
haha this is a good point. i will make it less vague.